kpi/tests/test_mock_data_exports.py

```diff
@@ -1028,12 +1052,16 @@ def test_export_latest_version_only(self):
         self.asset = Asset.objects.get(pk=self.asset.pk)
         self.asset.content['survey'] = new_survey_content
         self.asset.save()
+        # Retrieve the uid of last deployed version before deploying. Otherwise,
+        # it won't match with export data because data has been submitted prior
+        # this deployment.
+        version_uid = self.asset.latest_deployed_version_uid
```

Comment:
Thanks for updating all these tests :sweat_smile:
It's unfortunate that evidently there's not a single test within kpi that includes submissions made with more than one version of a form. That really makes me start to doubt the utility of these tests: kpi really only needs to test that the form schemas, export options, and submission stream convey properly to formpack; the vast majority of testing export content should be handled by formpack itself. I'll create an issue for this.

Reply:
#4254